contactForm.php

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+</head>
+
+<body>
+    <?php
+
+    $firstName = '';
+    $lastName = '';
+    $email = '';
+    $message = '';
+
+    if (isset($_POST['submit_btn'])) {
+        if (!empty(trim($_POST['firstName']))) {
+            $firstName = trim($_POST['firstName']);
+        } else {
+            echo '<span class="error">*First Name is Required</span> </br>';
+        }
+
+        if (!empty(trim($_POST['lastName']))) {
+            $lastName = trim($_POST['lastName']);
+        } else {
+            echo '<span class="error">*Last Name is Required</span></br>';
+        }
+
+        if (!empty(trim($_POST['email']))) {
+            $email = trim($_POST['email']);
+        } else {
+            echo '<span class="error">*Enter your Email</span><br/>';
+        }
+
+        if (!empty(trim($_POST['message']))) {
+            $message = trim($_POST['message']);
+        } else {
+            echo '<span class="error">*Enter your Message</span><br/>';
+        }
+    }
+
+    ?>
+
+
+    <form action="contactForm.php" method="post" id="contact">
+
+        <div class="name">
+
+            First Name:*<input type="text" name="firstName" id="firstName" <?= (isset($_POST['firstName']) ? 'value="' . $_POST['firstName'] . '"' : '') ?> required />
+
+
+            Middle Initial:<input type="text" name="middleName" id="middleName" <?= (isset($_POST['middleName']) ? 'value="' . $_POST['middleName'] . '"' : '') ?> />
+
+
+            Last Name:*<input type="text" name="lastName" id="lastName" <?= (isset($_POST['lastName']) ? 'value="' . $_POST['lastName'] . '"' : '') ?> required />
+        </div> <br />
+
+
+        <div class="phone">
+            <label for="phone">Phone Number*</br>Format:123-456-7890</label>
+            <input type="tel" id="tel" name="tel" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" <?= (isset($_POST['tel']) ? 'value="' . $_POST['tel'] . '"' : '') ?> />
+
+        </div><br />
+
+
+
+
```

Comment:
why all the empty space? Trim down all the empty lines in the file and you're good to go.